integrations/cloudflare.go

```diff
@@ -1,84 +1,77 @@
 package integrations
 
 import (
-	"time"
-
+	"fmt"
 	cf "github.com/cloudflare/cloudflare-go"
+	"time"
 )
 
-var api *cf.API
-
-type Record struct {
-	Id        string
-	Zone      string
-	Ttl       uint
-	DnsType   string
-	DnsData   string
-	Timestamp time.Time
-}
-
-type Zone struct {
-	Id      string
-	Name    string
-	Records []Record
-}
+const healthCheckFailed = "health check for cloudflare failed"
 
-func (z *Zone) AppendRecords(r ...Record) {
-	if z.Records == nil {
-		z.Records = []Record{}
-	}
-	for _, rr := range r {
-		z.Records = append(z.Records, rr)
-	}
-	return
-}
-
-type Zones []Zone
+var api *cf.API
 
-// InitAPI -- Configure token to cloudflare
-func InitAPI(apiToken string) (err error) {
+//InitApi
```

Comment:
Codacy found an issue: [comment on exported function InitCloudFlareAPI should be of the form "InitCloudFlareAPI ..."](https://app.codacy.com/gh/go-go-dojo/friendly-potato/pullRequest?prid=6165294)